packages/elements/package.json

```diff
@@ -106,6 +106,9 @@
       "optional": true
     }
   },
+  "optionalDependencies": {
+    "@rollup/rollup-linux-x64-gnu": "^4.9.6"
```

Comment:
This is a sign that something else is wrong, this shouldn't be here and not necessary. Every other package that uses `tsup` would run into it otherwise. The commited lock file in this PR must be wrong